src/pykiso/test_result/multi_result.py

```diff
@@ -0,0 +1,243 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Multi test result
+*****************
+
+:module: multi_result
+
+:synopsis: implements a multi test result class that run test for multiple
+    result classes.
+
+.. currentmodule:: multi_result
+"""
+from __future__ import annotations
+
+from types import TracebackType
+from typing import Any, List, Optional, TextIO, Tuple, Type, Union
+from unittest.case import _SubTest
+
+from pykiso.test_result.text_result import BannerTestResult
+from pykiso.types import ExcInfoType
+
+from ..test_coordinator.test_case import BasicTest
+from ..test_coordinator.test_suite import BaseTestSuite
+from .xml_result import XmlTestResult
+
+
+class MultiTestResult:
+    """Class that can take multiple test result classes and ran the
+    test for all the classes.
+    """
+
+    separator1 = "=" * 70
```

Comment:
Are these required?

Reply:
if yes 70 could be a constant.

Reply:
I actually don't need them anymore, has been removed.